salt/metalk8s/products/init.sls

```diff
@@ -0,0 +1,81 @@
+{% import_yaml "/etc/metalk8s/bootstrap.yaml" as bootstrap_config %}
+
+{% set products = bootstrap_config.products.metalk8s %}
+
+{% set base = '/srv/scality' %}
+{% set roots_file = '/etc/salt/master.d/99-metalk8s-roots.conf' %}
+
+{% for product in products %}
+{%   if salt['file.file_exists'](product) %}
+{%     set product_version = salt['metalk8s.product_version_from_iso'](product) %}
+{%     if not product_version %}
+{%         set product_path = product %}
+{%         set product_version = 'unknown' %}
+{%         set product_path_available = "- test: Product " ~ product ~ " available" %}
+Product {{ product }} available:
+  test.configurable_test_state:
+  - changes: false
+  - result: false
+  - comment: >
+      Unable to retrieve product version from given product file {{ product }},
+      assumed to be an ISO archive containing a "product.txt" file.
+
+{%     else %}
+{%         set product_path = base ~ '/metalk8s-' ~ product_version %}
+{%         set product_path_available = "- mount: Product " ~ product ~ " available at " ~ product_path %}
+Product path {{ product_path }} exists:
+  file.directory:
+  - name: {{ product_path }}
+  - mode: 0755
+  - makedirs: true
+
+Product {{ product }} available at {{ product_path }}:
+  mount.mounted:
```

Comment:
It would be nice to add `persist: True` to have the iso mounted automatically and to be resilient when a restart occur.

Reply:
Note that it's the default, but we want to be explicit

Reply:
fixed